modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -93,6 +93,32 @@ The following snippet configures an Ethernet interface that uses a dynamic IP ad
 # ...
 ----
 
+[id="virt-example-nmstate-IP-management-mac_{context}"]
+== Media Access Control (MAC) address
+
+You can use a MAC address to identify a network interface instead of relying on the name of the network interface. Every network interface has a unique MAC address, making a MAC address an ideal candidate for identifying a network interface. Consider that a network interface name can change for various reasons, such as an operating system configuration change. 
+
+Supported values for the `identifier` parameter include `name`, the default value, and `mac-address`. The `name` value applies a configuration to an interface that holds a specified interface name. The `mac-address` value for `identifier` indicates that a MAC address represents the identifier for a network interface and requires that you set a MAC address value for the `mac-address` parameter. 
+~
+[NOTE]
+====
+You can still specify a value for the `name` parameter but the `identifier: mac-address` means that a MAC address is the primary identifier for a network interface.
```

Comment:
Does it fall back to trying the named device if the MAC-addressed device is gone, or does the connection just break if the MAC-addressed device is down/gone?